src/test/java/web_pages/YandexHomePage.java

```diff
@@ -0,0 +1,68 @@
+package web_pages;
+
+import io.qameta.allure.Step;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import utils.Utils;
+
+import java.util.Properties;
+
+public class YandexHomePage {
+    WebDriver driver;
+
+    private String propertyPath = "src/test/resources/mail.properties";
+    private Properties properties = Utils.getProperties(propertyPath);
+
+    @FindBy(xpath = "//div[contains(@class, 'login-new-items')]/a")
+    private WebElement loginButton;
+
+    public YandexHomePage(WebDriver driver) {
+        this.driver = driver;
+        PageFactory.initElements(this.driver, this);
+    }
+
+    @Step("Login to site")
+    public void loginToSite(){
+        WebDriverWait wait = new WebDriverWait(driver, 10);
+        driver.get("https://yandex.by/");
+        loginButton.click();
+        driver.findElement(By.id("passp-field-login")).sendKeys(properties.getProperty("USER_NAME"));
```

Comment:
it is not a very cool approach, to use locators in the method body. Please select one of the widely used approaches (PageObject or PageFactory), cause in the current case your class is only called page but u did not receive any benefits from pageObject pattern.

Reply:
fixed